app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render 'errors', resource: @comment %>
+
+<%= form_for [@article, @comment] do |f| %>
+  <p>
+    <%= f.label :commenter %>
+    <%= f.text_field :commenter %>
```

Comment:
Jeżeli już komentujący dodał pierwszy komentarz, to można mu to pole automatycznie wypełniać.